### ISSUE TITLE
Antialiasing (MSAA) - Setting / 32xS [Combined: 2x2 SS + 8x MS] Doesn't work in Windows 11

### ISSUE THREAD

**Issue by kithylin** [Orbmu2k/nvidiaProfileInspector#165]
As said in the title: Antialiasing (MSAA) - Setting of "32xS [Combined: 2x2 SS + 8x MS]" Doesn't work at all in any game in Windows 11. I can re-boot to Windows 10 on the same computer and same hardware and it will work there but when I boot to Windows 11 it does not work. The setting of "16xS [Combined: 2x2 SS + 4x MS]" and all settings below that do work. It is only "32xS [Combined: 2x2 SS + 8x MS]" that does not work in Windows 11. When setting this setting of "32xS [Combined: 2x2 SS + 8x MS]" then loading a game all antialiasing is disabled resulting in no antialiasing at all.

**Comment by Squall-Leonhart** [comment]
Works fine on OpenGL.

**Comment by kithylin** [comment]
I guess I should add some clarity: Does not work in Windows 11 with DirectX titles. It does work with DirectX titles in windows 10. Just not in Windows 11.

**Comment by Squall-Leonhart** [comment]
Report it to nvidia, profile inspector can't do anything about driver bugs.

**Comment by Squall-Leonhart** [comment]
You are wrong, Inspector exposes and sets variables that are already present in the driver and its profile set.
